### PR TITLE
Greg/efund

### PR DESCRIPTION
**Pull request by GodGreg**
Done
created app routes to store all routes and groups attached to them. Cleans up space in app.js by loading the routes defined in appRoutes
Removed all API calls from app.js to get.js (clean up code)
Added spinner for applications table (in case it takes a while to load)
Added deployment scripts (one for QA and one for Prod)
Created Google Service Account
Created backend routes for Firebase admin service account
Populated user mgmt table with actual users 
Synced firebase with Mongo using a command defined in userManagement.js
Private route: Check if admin (with app routes) and check Expiry token.
Push to decision-io prod to have secure program access login
Push to new prod site also with secure login and the new applications
Updated applications to be the new emergency fund applications
Fixed some login css an

To do:
Edit user functionality
Create user functionality
Send to Brittany to test

### CODE REVIEW

File: src/Components/Admin/UserManagement.jsx [hunk 1]
```diff
@@ -31,7 +35,7 @@ const Wrapper = styled.div`
 `;
 
 // Moock data
-const users = new Array(30).fill(0).map((elem, index) => ({
+const users2 = new Array(30).fill(0).map((elem, index) => ({
```
Comment (ahmedhamodi) [inline src/Components/Admin/UserManagement.jsx:38]: Fixed array size of 30 - not sure if this is okay or not.

Comment (ahmedhamodi) [reply src/Components/Admin/UserManagement.jsx:38]: I also question the variable name. Why was it changed from users to users2? And was the usage updated throughout? (ie. users2 isn't used anywhere else in this file)

File: src/Components/DecisionCanvas/CanvasCard.jsx [hunk 2]
```diff
@@ -86,11 +86,14 @@ function CanvasCard({
   let rating = 0;
   let notes = [];
 
+  console.log(review)
```
Comment (ahmedhamodi) [inline src/Components/DecisionCanvas/CanvasCard.jsx:89]: Minor: unneeded console log

File: src/Components/DecisionCanvas/CanvasCard.jsx [hunk 3]
```diff
@@ -86,11 +86,14 @@ function CanvasCard({
...
   if (review != null) {
     rating = review.rating >= 0 ? review.rating : 0;
     notes = review.notes;
   }
 
+  console.log(rating)
```
Comment (ahmedhamodi) [inline src/Components/DecisionCanvas/CanvasCard.jsx:95]: Minor: unneeded console log

File: src/Components/Header/Header.css [hunk 4]
```diff
@@ -41,14 +41,14 @@ Make it a variable... how?
 
   /* H5/ Roboto Regular 24 */
   float: right;
-  font-family: Tungsten-Semibold;
+  font-family: Roboto;
```
Comment (ahmedhamodi) [inline src/Components/Header/Header.css:44]: The change to Tungsten was from this task: https://www.notion.so/uwblueprintexecs/Update-the-Arbitrium-logo-font-to-Tungsten-34b6b413ff5c4291a92282d8cd8802fb

Was there a reason it was changed back to Roboto?

Comment (brittanylau) [reply src/Components/Header/Header.css:44]: during QA we noticed that the font wasn't displaying properly (I think it was showing as Times New Roman), so we changed it back to Roboto as a quick fix to release the app. let's definitely try to get the Tungsten font back in though!

**Pull request merged**